esp-hal/Cargo.toml

```diff
@@ -1,19 +1,20 @@
 [package]
-name         = "esp-hal"
-version      = "0.15.0"
-edition      = "2021"
-rust-version = "1.76.0"
-description  = "Bare-metal HAL for Espressif devices"
-repository   = "https://github.com/esp-rs/esp-hal"
-license      = "MIT OR Apache-2.0"
-links        = "esp-hal"
+name          = "esp-hal"
+version       = "0.16.0"
+edition       = "2021"
+rust-version  = "1.76.0"
+description   = "Bare-metal HAL for Espressif devices"
+documentation = "https://docs.esp-rs.org/"
```

Comment:
Hmm I'm just thinking, shouldn't this be `https://docs.esp-rs.org/esp-hal`? What happens if we want to host esp-wifi docs here too? For example we most likely want to have `https://docs.esp-rs.org/book` and `https://docs.esp-rs.org/esp-wifi` and a bunch more. Do we know if the way we currently have this configured will work?

Reply:
Fun fact! This auto updates the book URL, and even esp-idf-hal docs to the new shiny URL :star_struck: automagically. Neat!

Reply:
Thats really cool! Only one small detail, could we add some redirections? For example in https://ferrous-systems.com/blog/announce-esp-training/#embedded-rust-on-espressif-for-advanced the "training material" link points to https://docs.esp-rs.org/espressif-trainings/ which is not properly redirected (I think there are already 3 redirections taking place there hahah). 

Don't think that this a blocker/stopper as I don't think this will cause too many broken links (and if it does, if we even gonna notice it), but it would be cool if we could fix this one at least.

Reply:
I think we have two options:

* Ask Ferrous to change their redirect
* We can create espressif-trainings repo that redirects to std-training

Reply:
I'll probably suggest the first one, again, this a nitpick and I dont think it affects much (or at all) and by no means it should delay the release.

Let me know if you want me to reach FS folks to update this.